content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,202 @@ title: Instrumentation
 weight: 3
 ---
 
-This guide will cover creating and annotating spans, creating and annotating metrics, how to pass context, and a guide to automatic instrumentation for JavaScript. This simple example works in the browser as well as with Node.js
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+If a `TracerProvider` is not created, either explicitly or via SDK APIs, then
+the OpenTelemetry APIs for tracing will use a no-op implementation and fail to
+generate data.
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+### Node.js
 
-function doWork() {
-  console.log('work...');
-  // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
-}
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-node
```

Comment:
sdk-node is an experimental package. I wouldn't recommend it to be the official website documentation without a disclaimer about that. I would also not remove the documentation which uses the stable packages

Reply:
A quirk to this is that the getting-started docs use this SDK as of this PR: https://github.com/open-telemetry/opentelemetry.io/pull/697

Should that get reverted as well?

Reply:
It is the easiest way to get started so I'm not sure. It isn't likely to cause problems, but it is very likely to have changes which break its interface which users may not appreciate if it is presented as the preferred solution. It might be better to have it in a different documentation section explicitly marked as experimental.

Reply:
I'll stick with `sdk-trace-node` for now and leave a note about how you can optionally use the `sdk-node` experimental package. When that package is no longer experimental we can update docs here.